### PR TITLE
Fix: Cleanup API info + `autoload=no` and  delete 'elementor_scheme_%'. [ED-3204]

### PR DESCRIPTION
**Pull request by iNewLegend**

### CODE REVIEW

File: core/upgrade/upgrades.php [hunk 1]
```diff
@@ -928,6 +929,14 @@ public static function _v_3_2_0_migrate_breakpoints_to_new_system( $updater, $in
 		return self::move_settings_to_kit( $callback, $updater, $include_revisions );
 	}
 
+	public static function _v_3_4_0_remove_old_elementor_scheme_() {
```
Comment (StyleShit) [inline core/upgrade/upgrades.php:932]:
```suggestion
	public static function _v_3_4_0_remove_old_elementor_scheme() {
```

Do you need this underscore?

**Review by StyleShit — CHANGES_REQUESTED** [review]

File: core/upgrade/upgrades.php (hunk 1 above)
Comment (iNewLegend) [reply core/upgrade/upgrades.php:932]: Yes, the same I did with another cleanup in another task. Because its removing a prefix.

**Review by StyleShit — APPROVED** [review]

**Review by matipojo — CHANGES_REQUESTED** [review]

File: core/upgrade/upgrades.php (hunk 1 above)
Comment (iNewLegend) [reply core/upgrade/upgrades.php]: Handled.

File: core/schemes/base.php [hunk 2]
```diff
@@ -21,6 +21,8 @@ abstract class Base {
 	 */
 	const LAST_UPDATED_META = '_elementor_scheme_last_updated';
 
+	const SCHEME_OPTION_PREFIX = 'elementor_scheme_';
```
Comment (matipojo) [inline core/schemes/base.php:24]: Revert changes for this file because you delete the option on upgrade

**Review by matipojo — CHANGES_REQUESTED** [review]

File: core/schemes/base.php (hunk 2 above)
Comment (iNewLegend) [reply core/schemes/base.php:24]: Let talk about this in private, I think I didn't understand the task correctly.

File: core/upgrade/upgrades.php [hunk 3]
```diff
@@ -928,6 +929,14 @@ public static function _v_3_2_0_migrate_breakpoints_to_new_system( $updater, $in
 		return self::move_settings_to_kit( $callback, $updater, $include_revisions );
 	}
 
+	public static function _v_3_4_0_remove_old_elementor_scheme() {
```
Comment (iNewLegend) [inline core/upgrade/upgrades.php:932]:
```suggestion
	public static function _v_3_5_0_remove_old_elementor_scheme() {
```

File: core/upgrade/upgrades.php [hunk 4]
```diff
@@ -928,6 +931,22 @@ public static function _v_3_2_0_migrate_breakpoints_to_new_system( $updater, $in
 		return self::move_settings_to_kit( $callback, $updater, $include_revisions );
 	}
 
+	public static function _v_3_5_0_remove_old_elementor_scheme() {
+		global $wpdb;
+
+		$key = Base::SCHEME_OPTION_PREFIX;
+
+		$wpdb->query( "DELETE FROM {$wpdb->options} WHERE option_name LIKE '{$key}%';" ); // phpcs:ignore
```
Comment (matipojo) [inline core/upgrade/upgrades.php:939]: @kobizz Please approve

**Review by matipojo — APPROVED** [review]

File: core/upgrade/upgrades.php (hunk 4 above)
Comment (kobizz) [reply core/upgrade/upgrades.php:939]: If there is an old site that has not been migrated from the scheme system to the global system yet, it won't be able to do it anymore.

Comment (matipojo) [reply core/upgrade/upgrades.php:939]: It is firstly upgraded to a kit, the upgrades are orders by the versions.

**Review by kobizz — APPROVED** [review]

**Review by matipojo — CHANGES_REQUESTED** [review]

**Review by matipojo — CHANGES_REQUESTED** [review]
Please pull from 3.5.0

**Pull request merged**